Users/abehenderson/CSX/filterArray.js

```diff
@@ -0,0 +1,28 @@
+// Higher Order Function
+function filterArray(array, callback) {
+    const newArray = [];
+    for (let i = 0; i < array.length; i += 1) {
+      if (callback(array[i])) newArray.push(array[i]);
```

Comment:
This line is fine; as your experience increases you might see it written as:
```
    callback(array[i]) && newArray.push(array[i]);
```

Reply:
Ok thanks!

Reply:
will that work for `return` ?

For instance, it doesn't work to do `(num % 2 === 0) ? return num` or even `(num % 2 === 0) ? return num : return null;` or `(num % 2 === 0) ? return num : 0;`

But can you use the `&&` to return `num`?  `(num % 2 === 0) && return num;`

Actually, I realize I'm asking you here, but I'll just go figure it out on my own :-)